recipes/lz-ani/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.1.1" %}
+
+{% set system = "x64_linux" %}       # [linux and x86_64]
+{% set system = "x64_mac" %}         # [osx and x86_64]
+{% set system = "arm64_linux" %}     # [linux and aarch64]
+{% set system = "arm64_mac" %}       # [osx and arm64]
+
+{% set sha256 = "8f97ab716f2fb23248b6be9cf22b8e5507649c32fb9ab910e0d4e04b579dc75b" %}  # [linux and x86_64]
+{% set sha256 = "b7745c3219671518e1be94e1875029f0efc44d01ba5a16efd038bba23eec1dfd" %}  # [osx and x86_64]
+{% set sha256 = "f00471c0796e682824780f4323301469ca656a032c244434bd7496750ac5101d" %}  # [linux and aarch64]
+{% set sha256 = "3638a0fb7e4de16d2298f337db649ff8882de046c8590c0a1d85ee16c437d7c1" %}  # [osx and arm64]
+package:
+  name: lz-ani
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+  - {{ pin_subpackage('lz-ani', max_pin='x.x.x') }}
```

Comment:
Are you sure about `"x.x.x"` ?
Please see the Pull Request template for its meaning: https://github.com/bioconda/bioconda-recipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md#instructions-for-avoiding-api-abi-and-cli-breakage-issues